src/ngraph/onnx_import/node.hpp

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#pragma once
+
+#include <ostream>
+#include "onnx.pb.h"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        class Graph;
+
+        class Node
```

Comment:
Is this class needed only to serialize to a stream?

Reply:
No, this is work in progress, the class will have significantly more to do :)